Check failed: Build Documentation / linux docs (push)

### HydroMT plugin UWBM version checks

- [x] I have checked that this issue has not already been reported.
- [x] I have checked that this bug exists on the latest version of HydroMT plugin UWBM.

### Reproducible Example

Run quarto-dev/quarto-actions/publish@v2
Run git config --global user.email "quarto-github-actions-publish@example.com"
From https://github.com/Deltares-research/hydromt_uwbm
 * branch            gh-pages   -> FETCH_HEAD
 * [new branch]      gh-pages   -> origin/gh-pages
Preparing worktree (new branch 'gh-pages')
branch 'gh-pages' set up to track 'origin/gh-pages'.
HEAD is now at df6812c Built site for gh-pages
[gh-pages 3cce327] Built site for gh-pages
 20 files changed, 312 insertions(+), 113 deletions(-)
 create mode 100644 assets/deltares-blue.svg
 create mode 100644 assets/logo.jpeg
 delete mode 100644 robots.txt
 create mode 100644 site_libs/bootstrap/bootstrap-320e1ef9edb861ce49b497581fe021d4.min.css
 delete mode 100644 site_libs/bootstrap/bootstrap-adc86e7bd9615c2685d0123ba2ce0656.min.css
 create mode 100644 site_libs/quarto-html/axe/axe-check.js
 rename site_libs/quarto-html/{quarto-syntax-highlighting-37eea08aefeeee20ff55810ff984fec1.css => quarto-syntax-highlighting-587c61ba64f3a5504c4d52d930310e48.css} (98%)
origin	https://github.com/Deltares-research/hydromt_uwbm (fetch)
origin	https://github.com/Deltares-research/hydromt_uwbm (push)
remote: Permission to Deltares-research/hydromt_uwbm.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/Deltares-research/hydromt_uwbm/': The requested URL returned error: 403
ERROR: Error
    at gitCmds (file:///opt/quarto/bin/quarto.js:118172:13)
    at eventLoopTick (ext:core/01_core.js:178:7)
    at async file:///opt/quarto/bin/quarto.js:154697:5
    at async withWorktree (file:///opt/quarto/bin/quarto.js:154855:5)
    at async Object.publish2 [as publish] (file:///opt/quarto/bin/quarto.js:154694:3)
    at async publishSite (file:///opt/quarto/bin/quarto.js:158615:38)
    at async publish7 (file:///opt/quarto/bin/quarto.js:158951:58)
    at async doPublish (file:///opt/quarto/bin/quarto.js:158898:7)
    at async publishAction (file:///opt/quarto/bin/quarto.js:158917:5)
    at async _Command.actionHandler (file:///opt/quarto/bin/quarto.js:158876:9)
Error: Process completed with exit code 1.

### Current behaviour

Documentation does not update.

### Desired behaviour

Documentation should update

### Additional context

_No response_

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -9,12 +9,14 @@ on:
       - tests/**
       - hydromt_uwbm/**
       - docs/**
+      - .github/workflows/docs.yml
   pull_request:
     branches: [master]
     paths:
       - tests/**
       - hydromt_uwbm/**
       - docs/**
+      - .github/workflows/docs.yml
 
 jobs:
   docs:

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -6,15 +6,15 @@ on:
   push:
     branches: [master]
     paths:
-      - tests/*
-      - hydromt_uwbm/*
-      - docs/*
+      - tests/**
+      - hydromt_uwbm/**
+      - docs/**
   pull_request:
     branches: [master]
     paths:
-      - tests/*
-      - hydromt_uwbm/*
-      - docs/*
+      - tests/**
+      - hydromt_uwbm/**
+      - docs/**
 
 jobs:
   docs:

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -17,21 +17,20 @@ on:
       - docs/*
 
 jobs:
-  # Build docs on Linux
-  Docs:
+  docs:
     timeout-minutes: 10
-    name: linux docs
     runs-on: ubuntu-latest
-    env:
-      DOC_VERSION: dev
     defaults:
       run:
         shell: bash -l {0}
-    concurrency:
-      group:  ${{ github.workflow }}-${{ github.ref }}
-      cancel-in-progress: true
+    permissions:
+      contents: write
+      pages: write
     steps:
       - uses: actions/checkout@v6
+        with:
+          fetch-depth: 0
+
       - uses: astral-sh/setup-uv@v7
       - uses: quarto-dev/quarto-actions/setup@v2
 

